### ISSUE TITLE
Focal Loss parameter setting skills for Yolov2 training？

### ISSUE THREAD

**Issue by hewumars** [marvis/pytorch-yolo2#38]
I set Focal Loss params：
 class_num=4, alpha=Variable(torch.ones(4,1)*0.25), gamma=2, size_average=False

use CE（4 classes）：
pedestrian：       recall：90.45%，precision：66.25%
car：                   recall：96.40%，precision：57.89%
bicycle：             recall：96.09%，precision：90.29%
tricycle：             recall：96.42%，precision：81.54%

use FL（4 classes）：
pedestrian：       recall：89.15%，precision：69.15%
car：                   recall：96.29%，precision：61.15%
bicycle：             recall：96.09%，precision：92.23%
tricycle：             recall：96.25%，precision：84.43%

Compare both：
    I hope the recall rather than the precision rise  using FL。e.g.  pedestrian （90.45% fall to 89.15%）
    What should I do? Is this result correct?

**Comment by 4F2E4A2E** [comment]
What is the question here exactly, can you please explain?

**Comment by hewumars** [comment]
@4F2E4A2E recall has descend，I wish to increase。

**Comment by 4F2E4A2E** [comment]
IMHumbleO you would have to increase / scale the number of the true positives in order to increase your recall - but don't call me on that one 😅 
https://en.wikipedia.org/wiki/Precision_and_recall
